codex-rs/app-server-test-client/src/main.rs

```diff
@@ -717,6 +722,37 @@ impl CodexClient {
         Ok(())
     }
 
+    fn handle_file_change_request_approval(
```

Comment:
This is maybe more accurate?

```suggestion
    fn approve_file_change_request(
```